addons/zookeeper/templates/cmpd.yaml

```diff
@@ -43,27 +44,63 @@ spec:
       serviceable: false
       writable: false
       votable: false
+  updateStrategy: BestEffortParallel
+  vars:
+    - name: ZOOKEEPER_POD_FQDN_LIST
+      valueFrom:
+        componentVarRef:
+          optional: false
+          podFQDNs: Required
+    - name: ZOOKEEPER_METRICS_MONITOR
+      value: {{ .Values.metrics.enabled | quote }}
+    - name: ZOOKEEPER_METRICS_PORT
+      value: {{ .Values.metrics.port | quote }}
+    - name: ZOOKEEPER_DATA_DIR
+      value: {{ .Values.zookeeper.dataDir }}
+    - name: ZOOKEEPER_DATA_LOG_DIR
+      value: {{ .Values.zookeeper.dataLogDir }}
+    - name: ZOOBINDIR
+      value: "/opt/bitnami/zookeeper/bin"
+    - name: ZOOBIN
+      value: "/opt/bitnami/zookeeper/bin"
+    - name: ZOO_ENABLE_AUTH
+      value: "yes"
+    - name: SERVICE_PORT
```

Comment:
This var seems useless.

Reply:
service_port needs to be preserved as Gemini will utilize this variable.

Reply:
please add a comment to the code.

Reply:
> please add a comment to the code.

done